org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/OurConsole.java

```diff
@@ -66,6 +66,8 @@
  * be displayed in red.
  * <p>
  * <b>Thread Safety:</b> Can be called only by the AWT event thread.
+ *
+ * @modified: Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
the evaluator now acts on the current state focused on the visualizer (temporal formulas can still be evaluated, but always from the focused state); this info is updated by the `VizGUI` and passed along to the `Computer`